packages/input-edit/src/components/InputEdit/InputEdit.js

```diff
@@ -1,39 +1,88 @@
-import React, {useEffect} from 'react'
+import React, {useEffect, useMemo} from 'react'
 import PropTypes from 'prop-types'
-import {Panel} from 'tocco-ui'
+import {scale, Typography} from 'tocco-ui'
+import styled from 'styled-components'
+import SplitPane from 'react-split-pane'
+import {actions} from 'tocco-app-extensions'
+import {intlShape} from 'react-intl'
 
 import InputEditTable from '../InputEditTable/InputEditTableContainer'
-import {FlexRow} from './StyledComponents'
 import InputEditPagination from '../InputEditPagination'
 import InputEditSearch from '../InputEditSearch'
+import {resizerStyle, StyledSplitPanelWrapper} from './StyledInputEdit'
+import InputEditInformation from '../InputEditInformation'
 
-const InputEdit = ({entityKey, initializeTable, initializeSearch}) => {
+const StyledSplitPane = styled(SplitPane)`
+  position: static !important;
+
+  .react-bs-container-body {
+    height: auto !important;
+    margin-bottom: ${scale.space(0)};
+  }
+
+  > div {
+    overflow-y: hidden;
+  }
+`
+
+const InputEdit = ({entityKey, initializeTable, initializeSearch, initializeInformation, inputDataForm, intl}) => {
   useEffect(() => {
     initializeTable()
     initializeSearch()
+    initializeInformation()
   }, [entityKey])
 
-  return <FlexRow>
-    <Panel.Wrapper key="search">
-      <Panel.Body>
-        <InputEditSearch/>
-      </Panel.Body>
-    </Panel.Wrapper>
-    <Panel.Wrapper key="list" isToggleable={false}>
-      <Panel.Body>
-        <InputEditTable/>
-      </Panel.Body>
-      <Panel.Footer>
-        <InputEditPagination/>
-      </Panel.Footer>
-    </Panel.Wrapper>
-  </FlexRow>
+  const actionDefinitions = useMemo(() => inputDataForm.children ? inputDataForm.children
+    .find(child => child.id === 'main-action-bar')
+    .children
+    .filter(child => child.componentType === 'action')
+    .map(child => {
+      child.scope = 'detail'
+      return child
+    }) : [], [inputDataForm])
+  const selection = actions.getSingleEntitySelection('Input', entityKey)
+
+  return <StyledSplitPane
+    defaultSize={300}
+    minSize={300}
+    resizerStyle={resizerStyle}
+    split="vertical">
+    <StyledSplitPanelWrapper key={'sidebar'}>
+      <StyledSplitPane defaultSize={300}
+        minSize={300}
+        resizerStyle={resizerStyle}
+        split="horizontal">
+        <StyledSplitPanelWrapper key={'search'}>
+          <Typography.H2>{intl.formatMessage({id: 'client.input-edit.sidebar.search'})}</Typography.H2>
```

Comment:
in this case you can use FormattedMessage component since you use localized text in jsx directly.
https://formatjs.io/docs/react-intl/components